src/manifest/mod.ts

```diff
@@ -264,6 +262,30 @@ export class SlackManifest {
     // This is set by default as false
     manifest.oauth_config.token_management_enabled = false;
 
+    // Required App Settings for run on slack apps
+    manifest.settings.org_deploy_enabled = true;
+
+    // App Home
+    // Default to messages enabled
+    manifest.features.app_home = {
+      messages_tab_enabled: true,
+      messages_tab_read_only_enabled: true,
+    } as ManifestAppHomeMessagesTabSchema;
+
+    // Allow App Home override values if provided in apphome
+    if (def.features?.appHome) {
+      const {
+        messagesTabEnabled,
+        messagesTabReadOnlyEnabled,
+      } = def.features.appHome;
+
+      if (manifest.features.app_home !== undefined) {
```

Comment:
With the defaulting now happening right above this, we can remove this if statement since it will never be undefined

Reply:
Yup! Removed it.